core/types/calls.cc

```diff
@@ -4386,6 +4444,7 @@ const vector<Intrinsic> intrinsics{
 
     {Symbols::Kernel(), Intrinsic::Kind::Instance, Names::proc(), &Kernel_proc},
     {Symbols::Kernel(), Intrinsic::Kind::Instance, Names::lambda(), &Kernel_proc},
+    {Symbols::Kernel(), Intrinsic::Kind::Instance, Names::raise(), &Kernel_raise},
```

Comment:
Since `Kernel#raise` and `Kernel#fail` are the same function (#5930), we should have an intrinsic for `Kernel#fail` as well?